sdk/src/main/scala/com/horizen/ClosedBoxesZendooMerkleTree.scala

```diff
@@ -55,10 +55,10 @@ class ClosedBoxesZendooMerkleTree(val statePath: String, val dbPath: String, val
     transaction.newBoxes().asScala.view.map(_.id()).map(bytesToFieldElement).toSet
 
   private def elementsCouldBeAdded(fieldElements: Set[FieldElement]): Boolean =
-    fieldElements.map(element => merkleTree.getPosition(element)).forall(merkleTree.isPositionEmpty)
+    fieldElements.map(fieldElement => merkleTree.getPosition(fieldElement)).forall(merkleTree.isPositionEmpty)
 
   private def elementsCouldBeRemoved(fieldElements: Set[FieldElement]): Boolean =
-    fieldElements.map(element => merkleTree.getPosition(element)).forall(element => !merkleTree.isPositionEmpty(element))
+    fieldElements.map(fieldElement => merkleTree.getPosition(fieldElement)).forall(element => !merkleTree.isPositionEmpty(element))
```

Comment:
in previous comment, I meant the name of `element` in `.forall(element => !merkleTree.isPositionEmpty(element))`.
we should rename it to `position`

Reply:
done